src/uwtools/resources/jsonschema/make-hgrid.jsonschema

```diff
@@ -0,0 +1,301 @@
+            },
+            "halo": {
+              "type": "integer"
+            },
+            "iend_nest": {
+              "items": {
+                "type": "integer"
+              },
+              "type": "array"
+            },
+            "istart_nest": {
+              "items": {
+                "type": "integer"
+              },
+              "type": "array"
+            },
+            "jend_nest": {
+              "items": {
+                "type": "integer"
+              },
+              "type": "array"
+            },
+            "jstart_nest": {
+              "items": {
+                "type": "integer"
+              },
+              "type": "array"
+            },
+            "lat_join": {
+              "type": "number"
+            },
+            "my_grid_file": {
+              "type": "string"
+            },
+            "nest_grids": {
+              "type": "integer"
+            },
+            "nlat": {
+              "items": {
+                "type": "integer"
+              },
+              "type": "array"
+            },
+            "nlon": {
+              "items": {
+                "type": "integer"
+              },
+              "type": "array"
+            },
+            "non_length_angle": {
+              "type": "boolean"
+            },
+            "nratio": {
+              "type": "number"
+            },
+            "nxbnds": {
+              "type": "integer"
+            },
+            "nybnds": {
+              "type": "integer"
+            },
+            "out_halo": {
+              "type": "integer"
+            },
+            "parent_tile": {
+              "items": {
+                "type": "integer"
+              },
+              "type": "array"
+            },
+            "refine_ratio": {
+              "items": {
+                "type": "number"
+              },
+              "type": "array"
+            },
+            "rotate_poly": {
+              "type": "boolean"
+            },
+            "shift_fac": {
+              "type": "number"
+            },
+            "simple_dx": {
+              "type": "number"
+            },
+            "simple_dy": {
+              "type": "number"
+            },
+            "stretch_factor": {
+              "type": "number"
+            },
+            "target_lat": {
+              "type": "number"
+            },
+            "target_lon": {
+              "type": "number"
+            },
+            "verbose": {
+              "type": "boolean"
+            },
+            "xbnds": {
+              "items": {
+                "type": "number"
+              },
+              "type": "array"
+            },
+            "ybnds": {
+              "items": {
+                "type": "number"
+              },
+              "type": "array"
+            }
+          },
+          "required": [
+            "grid_type"
+          ]
+        },
+        "execution": {
+          "$ref": "urn:uwtools:execution"
```

Comment:
If this is a serial / non-MPI executable (and since the prototype config in `docs/shared` doesn't include MPI info, I'm guessing it is), we should use `urn:uwtools:execution-serial` here, which is on `main` now (but was probably not when you wrote this).

Reply:
Yep, I forgot to leave a note that it was pending that update to `main` - I'll add it, thanks!